larray/core/array.py

```diff
@@ -5984,6 +5984,98 @@ def expand(v, length):
         chunks.append(self[axis.i[start:]])
         return concat(chunks, axis)
 
+    def drop(self, labels=None, indices=None, axis=None):
+        r"""Return array without some labels or indices along an axis.
+
+        Parameters
+        ----------
+        labels : scalar or Group
+            Label(s) or group to remove.
+        indices : int or list/array of ints, optional
```

Comment:
1. It is not clear if it is possible to use `lables` and `indices` in the same time. I know from the implementation of the method that the answer is no. I think it should be mentioned clearly that both arguments are exclusive in the doc of the method.
2. Given the examples below, I understand that `indices` can be an IGroup right?